README.md

```diff
@@ -5,3 +5,17 @@ from stdin and writes it to stdout. Can also be run as a web service by passing
 the `-addr` option.
 
 ![PARTY](/out.gif)
+
+# Install
+
+```
+$ go get github.com/disintegration/imaging
```

Comment:
it looks like the party-aas project is outside the gopath, so the go tool is confused cause it can't determine its full import path. you should put it under "$GOPATH/src/github.com/mediocregopher/party-aas", then it should work.